README.md

```diff
@@ -1,5 +1,8 @@
-Voronoi Godot integration
-=========================
+Voronoi Godot integration for Godot 4.x
+=======================================
+
+This fork upgrades the orginal [library](https://github.com/rakai93/godot_voronoi) to support Godot 4.x.
```

Comment:
I guess this should be re-written to exclude "this fork" before being merged back.